lib/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @fileoverview Shareable eslint plugin for use by eslint-config-silvermine
+ * @author Justin Hummel
+ */
+"use strict";
```

Comment:
This code doesn't match our eslint rules ... for example, all the double quotes in this file. Please make this plugin repo lint itself using our shared rules.